sklearn/inspection/tests/test_plot_partial_dependence.py

```diff
@@ -0,0 +1,325 @@
+import numpy as np
+from scipy.stats.mstats import mquantiles
+
+import pytest
+from numpy.testing import assert_allclose
+
+from sklearn.datasets import load_boston
+from sklearn.datasets import load_iris
+from sklearn.datasets import make_classification, make_regression
+from sklearn.ensemble import GradientBoostingRegressor
+from sklearn.ensemble import GradientBoostingClassifier
+from sklearn.linear_model import LinearRegression
+from sklearn.inspection import plot_partial_dependence
+
+
+boston = load_boston()
```

Comment:
it would be better to name it BOSTON.

Maybe this is time to use a fixture for the data here instead?